L/LASzip/build_tarballs.jl

```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder, Pkg
 
 name = "LASzip"
-version = v"3.4.3"
+version = v"3.4.3-1.6"
```

Comment:
Sigh, no, you have to stick to `X.Y.Z` only.  I guess I'm going to prohibit anything else completely, since this is just the second attempt in one day: https://github.com/JuliaPackaging/Yggdrasil/pull/5344#discussion_r953604740

Reply:
Should we have a PkgVersion (`pv"1.2.3"`) or similar to enforce such things?